STYLE-GUIDE.md

```diff
@@ -19,10 +19,28 @@ Words are important.  Pulumi strives to use language that is clear, harmonious,
 * Ensure that readers are able to scan the headings of a page and get an effective overview of the page's content.
 * Every page should have exactly one `h1`.
 * Headings levels should only increment one level at a time.  E.g., if your previous heading level was an `h2`, the next heading must be an `h2` or an `h3`, but not, e.g., an `h4` or `h5`.
+*  Include keywords relating to the content immediately following the heading in `h2` and deeper.
+*  Try to front-load your keywords into the sentence to indicate more immediately to the reader the subject matter of the following content.
+  * Alternate your topical keywords; visit an [online thesaurus](https://www.thesaurus.com/) if you are stuck.
+*  Try not to use repetitive headings (e.g., starting with the same word, using the same keyword).
+
+## Images
+
+* If it makes sense, include an image that supports your topic beneath the first `h2`. 
+  - Use a file name that matches the keywords in the `h1` or `h2`.
+  - Create an ALT tag that directly matches the file name for accessibility.
```

Comment:
hmm, can you explain more about this? our file names are often extremely short and without spaces. i do not think they are appropriate to use for the alt text. id suggest linking to https://www.w3.org/WAI/tutorials/images/informative/ to help guide people into writing good alt text.

Reply:
Yes, please look below at the examples associated with this best practice in rows 35 & 36 and 38 & 39. The recommendation includes using a better file naming convention and matching the file naming convention with the ALT tag. This is specifically "if it makes sense, include an image that supports your topic" and not every image.

Reply:
i do not think our file names should match the alt text. they do not have spaces, so screen readers will not announce them correctly. if you would like better file names for images i think that you should open up an issue that describes the naming conventions you want.

Reply:
Thank you for your input.

Reply:
ref:
https://www.w3.org/TR/WCAG20-TECHS/F30.html

Reply:
I'm going to +1 Susan on this. File names, by necessity, do not use spaces, but rather underscores or dashes, due to spaces breaking filesystems on Linux-based computers (i.e., if you use spaces in a file name, the site is built on a computer running a Linux operating system, and spaces in a file name has the potential to cause at minimum broken image links and at most failure of the system to be able to read anything from disk). We've got quite a few customers who value accessibility for various reasons, and we cannot break that need.

The examples you're pointing to in lines 35/36/38/39 don't directly match the file name as this line requires as the way it's written expects that you would include the underscores or dashes ("directly matches"). They also don't actually describe the image, which is problematic as it will cause site scan failures in Lighthouse and other site validation tools that check for accessibility. Alt text should describe the image if the image is critical for someone to understand the rest of the article.

```suggestion
  - Create an ALT tag for accessibility that includes enough information for someone who cannot view the image to get the same information. If possible, include all of the words in the file name, and ensure you give the intent, purpose, and meaning of the image in the text.
```

Reply:
Correct me if I'm wrong, but the file names on line 36 is david-flanagan-cloud-engineering-ted-talk.jpg and 39 is christian-nunciato-reflections-deploying-mern-digitalocean.gif which includes dashes and not spaces while the ALT is the human-readable version of the file name. The words are a direct match with the exception of spaces instead of dashes, one dash in the second file name to separate ideas, and an apostrophe s to denote ownership. 

I don't understand, both of you are trying to correct what is already correct in your comments. I agree with the added information, I was originally told to be brief, so I'm going to commit the wording change. So when I say directly match, I'm comparing the file name with the correct file naming protocol with the human spoken component that is not written like a file name.